mitodl/open-discussions
# Added UI for adding/deleting user websites

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #1398

#### What's this PR do?
Adds UI for adding/deleting user websites in the profile page

#### How should this be manually tested?
- Try adding websites in the profile edit page then deleting them
- After adding websites, look at the profile page (not in edit mode) and make sure the links all show up

#### Screenshots (if appropriate)
<img width="681" alt="ss 2018-11-29 at 01 43 47" src="https://user-images.githubusercontent.com/14932219/49223010-83bda080-f3ab-11e8-9eb4-814a5d81a2a5.png">
<img width="661" alt="ss 2018-11-28 at 16 20 38" src="https://user-images.githubusercontent.com/14932219/49223011-83bda080-f3ab-11e8-8ca7-2a002a862a3d.png">
<img width="667" alt="ss 2018-11-28 at 16 20 16" src="https://user-images.githubusercontent.com/14932219/49223012-84563700-f3ab-11e8-8d13-2afb645f1cd2.png">
